src/realtimecontroller/DefaultRealtimeController.ts

```diff
@@ -267,12 +267,26 @@ export default class DefaultRealtimeController implements RealtimeController {
       this.onError(e);
     }
   }
-
-  realtimeUnsubscribeFromVolumeIndicator(attendeeId: string): void {
-    try {
-      delete this.state.volumeIndicatorCallbacks[attendeeId];
-    } catch (e) {
-      this.onError(e);
+  realtimeUnsubscribeFromVolumeIndicator(
+    attendeeId: string,
+    callback?: (
+      attendeeId: string,
+      volume: number | null,
+      muted: boolean | null,
+      signalStrength: number | null,
+      externalUserId?: string
+    ) => void
+  ): void {
+    if (!callback) {
+      try {
+        delete this.state.volumeIndicatorCallbacks[attendeeId];
+      } catch (e) {
+        this.onError(e);
+      }
+    } else {
+      if(this.state.volumeIndicatorCallbacks[attendeeId]) {
+        this.state.volumeIndicatorCallbacks[attendeeId] = this.state.volumeIndicatorCallbacks[attendeeId].filter(f => f !== callback);
```

Comment:
This branch needs `try..catch`, too. It is vanishingly unlikely that anything would go wrong in this expression, but this class is written to be as bulletproof as possible — see the block comment at the top. Best to copy the patterns in the other removal methods.